engine/api/src/main/scala/pl/touk/nussknacker/engine/api/process/Source.scala

```diff
@@ -11,7 +11,7 @@ trait Source[+T] {
 
 }
 
-trait TestDataParserProvider[+T] { self: Source[T] =>
+trait SourceTestSupport[+T] { self: Source[T] =>
```

Comment:
Can you also change restriction of `TestDataGenerator` that it can be mixed only to `Source with SourceTestSupport`?

Reply:
Fixed.